### ISSUE TITLE
x1d files have SCI extensions

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#5344]
_Issue [JP-1706](https://jira.stsci.edu/browse/JP-1706) was created on JIRA by [James Davies](https://jira.stsci.edu/secure/ViewProfile.jspa?name=jdavies):_

The resulting `_x1d` files produced by the spec3 pipeline have SCI extensions with empty arrays and the following header keywords:

```
# HDU SCI in jw00625-o070_s00163_niriss_f090w-gr150c_x1d.fits:
XTENSION= 'IMAGE   '           / Image extension                                
BITPIX  =                    8 / array data type                                
NAXIS   =                    0 / number of array dimensions                     
PCOUNT  =                    0 / number of parameters                           
GCOUNT  =                    1 / number of groups                               
EXTNAME = 'SCI     '           / extension name                                 
MJD-BEG =    59004.98880295301 / [d] exposure start time in MJD                 
MJD-AVG =    59004.98992136655 / [d] exposure mid-point in MJD                  
MJD-END =    59004.99103978009 / [d] exposure end time in MJD                   
TDB-BEG =    59004.98975693891 / [d] TDB time of exposure start in MJD          
TDB-MID =    59004.99087538631 / [d] TDB time of exposure mid-point in MJD      
TDB-END =    59004.99199383371 / [d] TDB time of exposure end in MJD            
XPOSURE =              171.789 / [s] Effective exposure time                    
TELAPSE =            193.26186 / [s] Total elapsed exposure time                
```

There shouldn't be a SCI array in an `_x1d` file.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1706?focusedCommentId=484741#comment-484741):_

I'm sure the SCI extension is being created just because of the existence of those time-related keyword attributes in the data model.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1706?focusedCommentId=507708#comment-507708):_

Fixed by #5694
